src/app/pages/ArticlePage/ArticlePage.styles.ts

```diff
@@ -55,16 +55,22 @@ export default {
       gridColumn: '1 / span 12',
       paddingBottom: '2rem',
     }),
-  mainContent: ({ palette, spacings }: Theme) =>
-    css({
-      paddingBottom: `${spacings.TRIPLE}rem`,
+  mainContent:
+    (liteCTAShows: boolean) =>
+    ({ palette, spacings }: Theme) =>
+      css({
+        paddingBottom: `${spacings.TRIPLE}rem`,
 
-      '.continueReadingFocusedElement': {
-        outline: `${pixelsToRem(3)}rem solid ${palette.BLACK}`,
-        boxShadow: `0 0 0 ${pixelsToRem(3)}rem ${palette.WHITE}`,
-        outlineOffset: `${pixelsToRem(3)}rem`,
-      },
-    }),
+        [liteCTAShows
+          ? '> *:nth-child(n + 9):not(button)'
+          : '> *:nth-child(n + 8):not(button)']: {
+          ':focus-visible': {
```

Comment:
Just want to check but would anything without :focus-visible support i.e Safari < 15 or older android lose the keyboard focus indicator altogether and are we fine with this for the experiment?

Reply:
Probably would lose it if they don't support it. We use `focus-visible` in quite a few instances outside of this. It kinda suits the use case here better, going by MDN:

> The :focus-visible pseudo-class also matches the focused element, but only if the user needs to be informed where the focus currently is